libsrc/osic1p/cclear.s

```diff
@@ -22,7 +22,7 @@ _cclear:
         cmp     #0              ; Is the length zero?
         beq     L9              ; Jump if done
         sta     tmp1                                 
-L1:     lda     #$20            ; Blank - screen code
+L1:     lda     #' '
```

Comment:
I hoped it would earn me some bonus points for the rest of the review :jack_o_lantern: